### DISCUSSION TITLE
每次重启Obsidian后，所有插件恢复了之前默认设置，是什么原因？

### DISCUSSION THREAD

**Discussion by tabortao** [oldwinter/knowledge-garden#83] [提问]
向咨询下，为什么每次重启后，所有插件设置都会回复如初，是那里出了问题吗？

**Comment by oldwinter** [comment]
打开隐藏文件，查看根目录下的.obsidian文件夹里面的json格式的文件内容，在你通过obsidian软件界面更改配置的时候，以及重启obsidian的时候，是否被修改。我猜跟你使用的什么同步软件有关系。

**Reply by tabortao** [reply]
尝试看了，还是不行，我关闭所有第三方插件后，重新启动，第三方软件又打开了，太奇怪了。还是参照你的方案，自己进行搭建，学习下吧，虽然没解决，但是还是非常感谢。

**Reply by tabortao** [reply]
我以为是有啥插件，实现了**一键还原**导致的结果。
